infer/SigSuggestion.cc

```diff
@@ -175,6 +175,7 @@ void extractSendArgumentKnowledge(core::Context ctx, core::Loc bindLoc, cfg::Sen
         auto &t = typeAndOriginsOwner.emplace_back(make_unique<core::TypeAndOrigins>());
         t->type = arg.type;
         t->origins.emplace_back(core::Loc::none());
+        args.emplace_back(t.get());
```

Comment:
I have no idea why this wasn't here, and how we only just now noticed it.

The only reason why I thought to try this was because I saw that we weren't actually ever appending to this list, and  we did something similar 30 lines above:

https://github.com/sorbet/sorbet/blob/75c3c986fa4c6aca911d6226fe1b0f3beb81ec36/infer/SigSuggestion.cc#L150-L154